docs/_docs/maintaining/releasing-a-new-version.md

```diff
@@ -32,7 +36,7 @@ This updates the website's changelog, and pushes the versions in various other p
 
 It's recommended that you go over the `History.markdown` file manually one more time, in case there are any spelling errors or such. Feel free to fix those manually, and after you're done generating the website changelog, commit your changes.
 
-## Write a release post
+## Write a release post (`master` branch)
```

Comment:
This changes the anchor for this heading. How about using `{: #write-a-release-post}`?